src/spaces/graph.jl

```diff
@@ -113,7 +109,7 @@ function node_neighbors(
     else
         LightGraphs.all_neighbors
     end
-    neighborfn(model.space.graph, node_number)
+    Base.Generator(Int, neighborfn(model.space.graph, node_number))
```

Comment:
Why? `neighborfn` doesn't return Integers?

Reply:
It does, but not an iterator.